modules/e2eping/e2eping.js

```diff
@@ -126,8 +129,13 @@ class ParticipantWrapper {
         }
 
         if (numRequestsWithResponses >= this.e2eping.numRequests) {
-            console.log(`Measured RTT=${rtt} ms to participant ${this.id} (in
-                region ${this.participant.getProperty('region')}`);
+            logger.info(`Measured RTT=${rtt} ms to participant ${this.id} (in `
+                + `region ${this.participant.getProperty('region')}`);
+            this.clearIntervals();
+        } else if (totalNumRequests > 2 * this.e2eping.numRequests) {
+            logger.info(`Stopping e2eping for ${this.id} because we've sent `
+                + `${totalNumRequests} with only ${numRequestsWithResponses} `
+                + `responses.`)
```

Comment:
The log message doesn't match the condition?

Reply:
How so? It should log something like:
```
Stopping e2eping for ${this.id} because we've sent 11 with only 0 responses.
```

Reply:
These are not the same. The purpose of this check is to stop sending requests if we're not getting responses for some reason. 

this.e2eping.numRequests is essentially a constant (5) for the number of successful requests we want before we log the RTT measurement (I think it needs a better name, any suggestions?), so I didn't add it to the log message because it's not very interesting. `numRequestsWithResponses` is the number of responses we've got, which is interesting to log.